frontend/app/components/daterange-edit.js

```diff
@@ -26,6 +27,7 @@ export default class DaterangeEdit extends ApplicationComponent {
   @computed("entity.monthFrom")
   get selectedMonthFrom() {
     let selectedValue = this.entity.monthFrom;
+    this.setToAttribut("entity.monthTo", selectedValue);
```

Comment:
should be `setToAttribute` in English, make sure to change in all the places.

Reply:
Name of method has been changed